### PR TITLE
Add Highrise Blockchain Testnet

### PR DESCRIPTION
**Pull request by mculinovic**

### CODE REVIEW

**Comment by oyakh** [comment]
How can I add my wallet address

**Review by ligi — CHANGES_REQUESTED** [review]
The URL must be a IPFS CID

File: _data/chains/eip155-8184.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Highrise Blockchain Testnet",
+  "chain": "Highrise Blockchain Testnet",
+  "icon": "highrise_testnet",
+  "rpc": [
+    "https://subnets.avax.network/pocket-worlds/testnet/rpc",
```
Comment (najmahmd) [inline _data/chains/eip155-8184.json:6]: 0x7ED3ac6Ee8e905F2341eC035118a0F7c42c8A677

**Review by najmahmd — APPROVED** [review]

**Review by najmahmd — APPROVED** [review]

**Comment by mculinovic** [comment]
> The icon URL must be a IPFS URL (starting with ipfs://)

@ligi added IPFS url

**Comment by mculinovic** [comment]
@ligi just a reminder for this when you have time to check it out

**Pull request closed without merging**